hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestEndpoints.java

```diff
@@ -635,22 +635,44 @@ public void testGetFileCounts() throws Exception {
     given(omKeyInfo3.getDataSize()).willReturn(1000L);
 
     OMMetadataManager omMetadataManager = mock(OmMetadataManagerImpl.class);
-    TypedTable<String, OmKeyInfo> keyTable = mock(TypedTable.class);
+    TypedTable<String, OmKeyInfo> keyTable1 = mock(TypedTable.class);
+    TypedTable<String, OmKeyInfo> keyTable2 = mock(TypedTable.class);
```

Comment:
Instead of using 1 and 2 everywhere, let's be a little more clear and use legacy and fso

Reply:
Thanks for the suggestion, i'll make the necessary changes!